sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -542,9 +542,9 @@ case class UnresolvedOrdinal(ordinal: Int)
 /**
  * Represents unresolved aggregate with having clause, it is turned by the analyzer into a Filter.
```

Comment:
@xuanyuanking Could we please update this comment to reflect the new semantics of this node ?

Reply:
Thanks, done in f2a2ad1.